src/main/resources/assets/pneumaticcraft/patchouli_books/book/zh_cn/entries/intro.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "气动机械",
+  "icon": "create:cogwheel",
+  "category": "pneumaticcraft:compressed_creativity",
+  "advancement": "create:andesite_alloy",
+  "priority": "true",
+  "pages": [
+    {
+      "type": "text",
+      "text": "你可曾想过：$(o)“气动机器真是酷，但总该能把它们变得酷上加酷。”$()不必担心，现在你就能体验气动工艺和机械动力联动的内容了！"
+    },
+    {
+      "type": "spotlight",
+      "item": "compressedcreativity:rotational_compressor",
+      "text": "$(l:pneumaticcraft:rotational_compressor)旋转压缩机$()是一种使用机械动力的旋转力产生压力的压缩机。"
+    },
+    {
+      "type": "spotlight",
+      "item": "create:copper_backtank",
+      "text": "The $(l:pneumaticcraft:copper_backtank)Copper Backtank$() received additional functionality, having pneumatic capabilities in both Item and Block form."
```

Comment:
Seems to have missed several terms in this file though...

Reply:
Strange, seemes this file didn't get updated correctly...
Gonna try to fix it.

Reply:
I guess it's OK now...